modules/appeals_api/app/controllers/appeals_api/appealable_issues/v0/appealable_issues_controller.rb

```diff
@@ -7,25 +7,73 @@ class AppealableIssuesController < AppealsApi::V2::DecisionReviews::ContestableI
     include AppealsApi::OpenidAuth
     include AppealsApi::Schemas
 
+    # This validation happens in #validate_params now; remove this once inheritance relationship is gone:
+    skip_before_action :validate_json_schema
+
     SCHEMA_OPTIONS = { schema_version: 'v0', api_name: 'appealable_issues' }.freeze
 
     OAUTH_SCOPES = {
       GET: %w[veteran/AppealableIssues.read representative/AppealableIssues.read system/AppealableIssues.read]
     }.freeze
 
+    def index
+      get_appealable_issues_from_caseflow
+
+      if caseflow_response_has_a_body_and_a_status?
+        render_response(caseflow_response)
+      else
+        render_unusable_response_error
+      end
+    end
+
     def schema
-      render json: AppealsApi::JsonSchemaToSwaggerConverter.remove_comments(headers_schema)
+      render json: AppealsApi::JsonSchemaToSwaggerConverter.remove_comments(form_schemas.schema('PARAMS'))
     end
 
     private
 
-    def header_names = headers_schema['definitions']['appealableIssuesIndexParameters']['properties'].keys
+    def validate_params = form_schemas.validate!('PARAMS', params.to_unsafe_h)
```

Comment:
I used the `params.json` schema to validate the new `icn` and `receiptDate` parameters, but also to replace two validations that previously happened in code: (1) making sure `decisionReviewType` is valid and (2) making sure `benefitType` is correct and present when required. See the original validations in the [parent controller here](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/appeals_api/app/controllers/appeals_api/v2/decision_reviews/contestable_issues_controller.rb#L126-L147). For this reason, the formatting of the error responses has changed in a few places - you can see how in the generated `swagger.json` file. I've also added some more specs for this v0 controller to make sure the validations are still happening correctly.